google/cloud/spanner_dbapi/connection.py

```diff
@@ -584,10 +633,14 @@ def connect(
     pool=None,
     user_agent=None,
     client=None,
+    buffer_ddl_statements=False,
     route_to_leader_enabled=True,
 ):
     """Creates a connection to a Google Cloud Spanner database.
 
+    :type buffer_ddl_statements: bool
+    :param buffer_ddl_statements:
```

Comment:
This needs a description of what it does, and when you should enable/disable it.

Reply:
Added